### PR TITLE
Move TObject2Json to QualityControl repo

### PR DESCRIPTION
**Pull request by awegrzyn**

### CODE REVIEW

**Comment by awegrzyn** [comment]
@vladimirkosmala Could you take look whether it provides same output?

File: Framework/src/TObject2Json.cxx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+void TObject2Json::start()
+{
+  while(1) {
+    // Wait for next request from client inside a zmq message
+    zmq_msg_t messageReq;
+    zmq_msg_init(&messageReq);
+    int size = zmq_msg_recv(&messageReq, mZeromqSocket, 0);
+    if (size == -1) {
+      throw std::runtime_error("Unable to receive zmq message: "s + zmq_strerror(zmq_errno()));
```
Comment (Barthelemy) [inline Framework/src/TObject2Json.cxx]: the process will die each time it is unable to receive. Is it something that might happen often / easily ?

Comment (vladimirkosmala) [reply Framework/src/TObject2Json.cxx]: Not often, only on bad conditions.
If it happens the process manager should restart it.
On special cases when it happens often, the exception should be handled to recover. Will see on stress tests very soon.

**Comment by awegrzyn** [comment]
@dberzano Could you take a look at the builder? It fails due to `gzip: stdin: not in gzip format`

**Comment by dberzano** [comment]
Unfortunately there's nothing I can do. Sourceforge is intermittently down, we've been having similar problems since this morning... updates here

> https://twitter.com/sfnet_ops

**Comment by awegrzyn** [comment]
Ok I see, what about previous fails have no content? When I click to preview the Full log it says `404 Not Found`.

**Comment by dberzano** [comment]
This is a cache delay in our web frontend. You'll find it now.

**Comment by awegrzyn** [comment]
@Barthelemy Is everything fine by you?

**Comment by Barthelemy** [comment]
@awegrzyn yes, it is. Please remove the "WIP" if it is ready for merge.

**Comment by awegrzyn** [comment]
Ok, I'll test it with @vladimirkosmala on Monday and then merge it.

**Pull request merged**